cli/src/commands/test-validator/index.ts

```diff
@@ -80,29 +80,43 @@ class SetupCommand extends Command {
       required: false,
       default: "127.0.0.1",
     }),
+    stop: Flags.boolean({
+      description: "Stops the test validator and dependent processes.",
+      required: false,
+      default: false,
```

Comment:
It's not critical, but I think it makes sense to add the `exclusive: [<all other available flags>]` parameter or even make this a standalone command.

Reply:
user can pass eg `--stop --skip-prover` to keep those alive

Reply:
gonna add to 'description' for clarity